Missing db.csv

Nikoleta-v3 opened the pull request:
Closes #7

A error occured when running the script when the db.csv file does not exists.
A function to create an empty db.csv was implemented and the used the existed
read_db function to read in the empty csv.

Also PEP8. 2 blank lines between the methods according to pycharm.

Commit:
Missing db.csv

Closes #7

A error occured when running the script when the db.csv file does not exists.
A function to create an empty db.csv was implemented and the used the existed
read_db function to read in the empty csv.

Also PEP8. 2 blank lines between the methods according to pycharm.

marcharper commented on update_fingerprints.py:
````diff
@@ -133,8 +149,11 @@ def main(turns, repetitions):
 fp.plot()
 ```
     """.format(version, turns, repetitions)
+    try:
````
Just a style comment -- I'd recommend splitting the read and create functions, or have the read function create the file if it doesn't exist. It's better IMO to have a function that creates a file if it doesn't exist and a separate function that reads.

Nikoleta-v3 replied:
Thank you for the tip @marcharper. Sorry for the delay I will take care of it now.

Commit:
seperating reading and creating

Seperated the functions read_db and create_db.The script
now tries to read if it fails it will create the file and them read
it again.

drvinceknight commented:
Merging :+1: Thanks Nik.
